packages/nuxt/src/app/composables/url.ts

```diff
@@ -2,9 +2,12 @@ import { getRequestURL } from 'h3'
 import { useRequestEvent } from './ssr'
 
 /** @since 3.5.0 */
-export function useRequestURL () {
+export function useRequestURL (opts?: {
+  xForwardedHost?: boolean
+  xForwardedProto?: boolean
+}) {
```

Comment:
we could probably use the type directly from `getRequestURL` in this case given that it's a pure wrapper.

Reply:
updated 👍